project 2/styles.css

```
@@ -0,0 +1,154 @@
+* {
+    box-sizing: border-box;
+}
+
+h1 {
+    font-family: 'Big Shoulders Display', cursive;
+    font-weight: 900;
+    text-align: center;
+    font-size: 24.87pt;
+    letter-spacing: 1px;
+    margin-top: 40px;
+    color: #32302A;
+}
+
+
+ 
+ ul.info {
+    font-family: 'Big Shoulders Display', cursive;
+    font-weight: 800;
+    font-size: 12pt;
+    border-top: 4px solid #CCC1A9;
+    border-bottom: 4px solid #CCC1A9;
+    border-left: 4px solid #CCC1A9;
+    display:inline-block;
+    list-style: none;
+    padding-left: 0;
+    margin-top: 1px;
+    margin-right: 15px;
+}
+li.info   {
+    border-right: 4px solid #CCC1A9;
+    display: inline-block;
+    padding: 0px 23px;
```

If you were writing this rule, you don't add any units (px, % or anything like that) to zero measurements.
```suggestion
    padding: 0 23px;
```
But even better would be to skip assigning the zero value and just write the code you actually need:
```suggestion
    padding-left: 23px;
    padding-right: 23px;
```